contracts/NFTStakingManager.sol

```diff
@@ -47,6 +47,7 @@ struct ValidationInfo {
   uint32 licenseCount;
   uint32 lastUptimeSeconds;
   uint32 lastSubmissionTime;
+  uint32 lastSubmissionEpoch;
```

Comment:
why store this? can't we calculate it?

Reply:
We don't set the submissionTime if they don't pass the uptime check. but I saw that in a later issue they comment that maybe we want to always set the submissionTime. I'm going to think about how we're calculating the uptime once more and then decide if we need this variable or not